bindings/java/pom.xml

```diff
@@ -76,7 +96,7 @@
             <configuration>
               <target>
                 <mkdir dir="${basedir}/resources" />
-                <exec executable="cp" dir="${basedir}/resources"
+                <exec executable="${cmd}" dir="${basedir}/resources"
```

Comment:
please use mvn native way to copy files, not platform dependant way.

https://maven.apache.org/plugins/maven-resources-plugin/examples/copy-resources.html

Reply:
Didn't notice you commented. Working on it now!

Reply:
@miked-mellanox 
Maven resources plugin is broken when trying to copy symlinks (*.so):
- In 2.7 it copies both the link and the destination of link (i.e. both .so and .so.0.0.0 will be copied).
- In 3.0.x it copies the link as a link (a broken one).

Not sure yet how should I solve it. I'll try to figure it out next week.

Reply:
do you mean link after copy pointing to the wrong location?

Reply:
Yes